### PR TITLE
Add option to only emit warnings (#51)

### LINKED ISSUE #51: Option to keep webpack build alive on error
Hey,
is there any way to keep the webpack build alive, even if a stylelint error has been thrown?

In https://www.npmjs.com/package/stylelint-webpack-plugin there's an option `emitErrors` which I believe turns errors into warnings, allowing the webpack build to finish.

Maybe I'm missing something here, but I think this could be very helpful.

### PR DESCRIPTION
**Pull request by ugrupp**
Fixes #51 
Spec resembles eslint: https://github.com/webpack-contrib/eslint-loader#errors-and-warning

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,10 @@ module: {
 }
 ```
 
+### emitWarning (default: `false`)
+
+Always return warnings if this option is set to `true`. If you're using hot module replacement, you may wish to enable this in development, or else updates will be skipped when there's an stylelint error.
```
Comment (emilgoldsmith) [inline README.md]: `when there's a stylelint error`, just a small typo to fix (`an` -> `a`)

**Comment by ugrupp** [comment]
Hey, thanks for the typo hint. I've also added a test now.

**Comment by emilgoldsmith** [comment]
Hey @ugrupp sorry for taking so long to publish, life got in the way, but it's live now :). I'll also invite you as a collaborator, thanks a lot for your initiative! :)

**Comment by ugrupp** [comment]
No problem, thanks a lot! :)

**Pull request merged**